src/prob/pf.jl

```diff
@@ -504,15 +506,21 @@ end
 Builds the solution dict.
 """
 function build_solution(pfd::PowerFlowData, Uv::Vector{Complex{Float64}})
-    solution = Dict{String, Any}("bus"=>Dict{String, Any}())
+    solution = Dict{String, Any}("bus"=>Dict{String, Any}(), "load"=>Dict{String, Any}(), "gen"=>Dict{String, Any}())
     for (id, bus) in pfd.data_math["bus"]
         ind = bus["index"]
         solution["bus"][id] = Dict{String, Any}()
         v = Dict(t=>_get_v(pfd, Uv, (ind,t)) for t in bus["terminals"])
         solution["bus"][id]["vm"] = Dict("$t"=>abs.(v[t]) for t in bus["terminals"])
         solution["bus"][id]["va"] = Dict("$t"=>angle.(v[t]) for t in bus["terminals"])
     end
-
+    for (ns, cc_func, ccd_func, comp_type, id) in pfd.cc_ns_func_pairs
+        v_bt = [solution["bus"]["$busid"]["vm"]["$t"] for (busid, t) in sort(ns)]
+        ccd = ccd_func(v_bt)
+        solution[comp_type][id] = Dict{String, Any}()
+        solution[comp_type][id]["ccp"] = real.(ccd)
```

Comment:
for loads, this should be "cdr" and "cdi" I believe, for gens "cgr" and "cgi"? for branches "cr" and "ci"?